lib/widget/carousel.dart

```diff
@@ -288,7 +299,36 @@ class CarouselState extends WidgetState<Carousel> with TemplatedWidgetState {
     );
   }
 
+  Widget? _buildIndicatorWidget(ScopeManager? scopeManager) {
```

Comment:
Can we have only one buildWidget?

Reply:
Do you mean to have one build method for creating the indicator? @snehmehta 
Okay, I'll combine both into a single method.

Reply:
@snehmehta I updated the code and can you check the `_buildIndicatorWidget` method?